### PR TITLE
Replaced the build id 1364869749170769920 with 1380606326060617728 in [27]

### PR DESCRIPTION
**Pull request by suppathak**
## Related Issues and Dependencies
Solved the bug issue #403 . Replaced the build id with the existing one from the list of builds. Refer to cell 27 in the notebook. 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements
Just changed the build id [Cell 27].

### CODE REVIEW

File: notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb [hunk 1]
```diff
@@ -92,7 +92,7 @@
     {
```
Comment (chauhankaranraj) [inline notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb:92]: <p>Should this number in the statement be changed to <code>&gt;98%</code> or <code>~100%</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (chauhankaranraj) [inline notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb:92]: <p>This conclusion doesn't seem valid anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (chauhankaranraj) [inline notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb:92]: <p>Same comment as above - this conclusion needs to be updated as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (chauhankaranraj) [inline notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb:92]: <p>So IIUC the issue was that build id <code>1364869749170769920</code> doesn't exist anymore and that's why running this cell gave an error. Can we reliably say build id <code>1380606326060617728</code> will exist on GCS couple of months from now? Basically, is there a way to <em>not</em> hardcode the build id and use something like <code>sorted(build_events_data.keys())[0]</code>?  WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (MichaelClifford) [reply notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb:92]: <p>+1</p>

Comment (suppathak) [reply notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb:92]: Made changes according to the feedback. Thanks :) @chauhankaranraj.

Comment (suppathak) [reply notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb:92]: @chauhankaranraj , I changed the percent value to 100%, as seen in the output of the code.

**Review by Shreyanand — COMMENTED** [review]
@suppathak Good catch! For the markdown with changed conclusions, you could either convert it to print statements with variables, something like `print(f"the mean is {np.mean(arr)}"`. Or, `from IPython.display import Markdown as md; md(str(np.mean([5,6,7])))`

For a really clean solution, we could try this [extension](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/python-markdown).

**Comment by suppathak** [comment]
> @suppathak Good catch! For the markdown with changed conclusions, you could either convert it to print statements with variables, something like `print(f"the mean is {np.mean(arr)}"`. Or, `from IPython.display import Markdown as md; md(str(np.mean([5,6,7])))`
> 
> For a really clean solution, we could try this [extension](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/python-markdown). Let me know if any of those seem good or if there are better alternatives :)

Thank you @Shreyanand . I have applied `md(statement)` format. I have also realized that some other statements, that stated the percentage value, have also changed. hence made the changes accordingly. Please have a look.
 
Also, I have tried to resolve the pre-commit error. However, after several try, I kept getting this flake8-nb error, `N813 camelcase 'Markdown' imported as lowercase 'md'`, which I could not resolve. If anybody have some experience on it, I would appreciate your feedback.

File: notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb [hunk 2]
```diff
@@ -311,67 +362,129 @@
...
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "The above shows us that the there are about 40% of the artifacts dirs that have 6 items and 20% that have 7 or 8 (but it does not account for different combinations) "
+    "Markdown('The above shows us that there are about {}% of the artifacts dirs '\n",
+    "         'that have {} items and {}% that have {} items '\n",
+    "         'and so on.'.format(round(cent2.iloc[0,1]*100,2),cent2.iloc[0,0],round(cent2.iloc[1,1]*100,2),cent2.iloc[1,0]))"
```
Comment (chauhankaranraj) [inline notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb:467]:
```suggestion
    "         'and so on (but it does not account for different combinations).'.format(round(cent2.iloc[0,1]*100,2),cent2.iloc[0,0],round(cent2.iloc[1,1]*100,2),cent2.iloc[1,0]))"
```

**Pull request merged**